### PR TITLE
Add Approval Management docs

### PR DESCRIPTION
**Pull request by shuvyA**
image

### CODE REVIEW

File: docs/embeddable-uis/element/approval-management.mdx [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+title: Approval Management
+---
+
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Approval Management Element
+
+<span className="subtitle-custom">
+The Approval Management Element is a Frontend component that shows you Operation Approvals. Easily approve or deny Operation Approvals.
```
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:12]: Need a better introduction here, maybe add link to operation approval doc

**Review by maya-barak — CHANGES_REQUESTED** [review]
more info missing

File: docs/embeddable-uis/element/approval-management.mdx [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+<br/>
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/approval-management.png" alt="approval-management-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+The Approval Management Element allows users with an Approver role to manage pending requests. Approver roles are assigned.
```
[the remark this answers is not in the document]
Comment (maya-barak) [reply docs/embeddable-uis/element/approval-management.mdx:24]: "Approver roles are assigned."- what is the meaning ?

**Review by RazcoDev — APPROVED** [review]

**Review by maya-barak — COMMENTED** [review]
Maybe change "reviewer" role to "Operation Approval Reviewer" or to "_reviewer_"

File: docs/embeddable-uis/element/approval-management.mdx [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+<br/>
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/approval-management/approval-management.png" alt="approval-management-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+The Approval Management Element allows users with role "Operation Approval Reviewer" to manage pending requests efficiently. The element displays a list of pending requests, and the reviewer can approve or deny them. The element is designed to be embedded in your application, and it is fully customizable to fit your application's look and feel.
```
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:24]: missing explanation about the connection to the operation approval element + add link to the relevant doc (explain that the reviewer role created at the operation element and the developer need to assign the user to this role, also after approve the user who ask the request will assigned to the approver role)

Comment (maya-barak) [reply docs/embeddable-uis/element/approval-management.mdx:24]: missing explanation about the webhook

File: docs/embeddable-uis/element/approval-management.mdx [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/approval-management/approval-management.png" alt="approval-management-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+The Approval Management Element allows users with role "Operation Approval Reviewer" to manage pending requests efficiently.
+This element based on [Operation Approval element](/embeddable-uis/element/operation-approval), reviewer and approval roles both will be created on the operation approval element. For view the operation approvals that have been created you will need to assigned the relevant user to the reviewer role.
```
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:25]: reviewer and approved roles both will be created

File: docs/embeddable-uis/element/approval-management.mdx [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+
+After the request will approve, the requesting user will be automatically assigned to the approve role (on the relvent resource instance for his request).
```
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:27]: will be automatically assigned to the approved role

**Review by maya-barak — APPROVED** [review]
LGTM- fix the approve role

**Pull request merged**